docs/platform/howto/_manage-org-vpc-peering-azure.md

```diff
@@ -0,0 +1,426 @@
+
+Establishing a peering connection between an Aiven organization VPC and an Azure VNet
+requires creating the peering both from the VPC in Aiven and from the VNet in Azure.
+
+To establish the peering from Aiven to Azure, the Aiven Platform's
+[Active Directory application object](https://learn.microsoft.com/en-us/azure/active-directory/develop/app-objects-and-service-principals)
+needs permissions in your Azure subscription. Because the peering is between different AD
+tenants (the Aiven AD tenant and your Azure AD tenant), another application object is
+needed for your Azure AD tenant to create the peering from Azure to Aiven once granted
+permissions to do so.
+
+## Prerequisites
+
+- [Manage organization networking](/docs/platform/concepts/permissions#organization-permissions)
+  permissions for the Aiven Platform
+- Azure account with at least the application administrator role
+- [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/?view=azure-cli-latest) and,
+  optionally, the [Microsoft Azure portal](https://portal.azure.com/#home)
+- Access to the [Aiven Console](https://console.aiven.io/)
+- [Aiven CLI](/docs/tools/cli) installed
+- Two VPCs to be peered: an
+  [organization VPC](/docs/platform/howto/manage-organization-vpc#create-an-organization-vpc)
+  in Aiven and a VNet in your Azure account
+
+## Set up permissions in Azure
+
+### Azure app object permissions
+
+1. Log in with an Azure admin account using the Azure CLI:
+
+   ```bash
+   az account clear
+   az login
+   ```
+
+   This should open a window in your browser prompting to choose an Azure
+   account to log in with.
+
+   :::tip
+   If you manage multiple Azure subscriptions, also configure the Azure CLI
+   to default to the correct subscription for the subsequent commands. This
+   is not needed if there's only one subscription:
+
+   ```bash
+   az account set --subscription SUBSCRIPTION_NAME_OR_ID
+   ```
+
+   :::
+
+1. Create an application object in your AD tenant using the Azure CLI:
+
+   ```bash
+   az ad app create                         \
+     --display-name "NAME_OF_YOUR_CHOICE"   \
+     --sign-in-audience AzureADMultipleOrgs \
+     --key-type Password
+   ```
+
+   This creates an entity to your AD that can be used to log into multiple
+   AD tenants ( `--sign-in-audience AzureADMultipleOrgs` ), but only the
+   home tenant (the tenant the app was created in) has the credentials to
+   authenticate the app.
+
+   :::note
+   Save the `appId` field from the output. It will be referred to as `$user_app_id`.
+   :::
+
+1. Create a service principal for your app object to the Azure subscription that the VNet
+   to be peered is located in:
+
+   ```bash
+   az ad sp create --id $user_app_id
+   ```
+
+   This creates a service principal to your subscription that may be given
+   permissions to peer your VNet.
```

Comment:
```suggestion
   This creates a service principal in your subscription, which can be 
   assigned permissions to peer your VNet.
```
Maybe something like this? I feel the preposition "to" in this context is incorrect.